shared/scss/popup.scss

```diff
@@ -457,68 +461,6 @@ body {
                 font-size: 13px;
                 padding: 7.5px 20px 15px 20px;
             }
-
-            .site-info__trackers__company-list {
-                overflow-y: auto;
-                height: 230px;
-                padding: 0 20px 60px 20px;
-
-                    li {
-
-                        &.is-empty {
-                            padding: 15px 0 120px 0;
-                        }
-
-                        &:first-child {
-                            padding-top: 7.5px;
-                        }
-
-                        .site-info__tracker__icon {
-                            @include icon_display();
-                            @include tracker_network_icons_subview();
-
-                            width: 22px;
-                            height: 22px;
-                            margin-top: -6px;
-                            margin-left: -4px;
-                            margin-bottom: 8px;
-                            overflow: hidden;
-                            line-height: 100px;
-                            background-size:contain;
-                        }
-
-                        &:last-child {
-
-                            .site-info__trackers__company-list__url-list {
-                                margin-bottom: 60px;
-                            }
-                        }
-                    }
-
-                .site-info__trackers__company-list__url-list {
-
-                    li {
-                        padding-top: 7.5px;
-                        font-size: 13px;
-                        color: $color--grey;
-                        position: relative;
-
-                        .url {
-                            display: block;
-                            width: 65%;
-                            text-overflow: ellipsis;
-                            overflow: hidden;
```

Comment:
Max width was already here. It's just none of those styles were applying because this was nested into `.site-info--details`, which wasn't happening in the new view.

I just copied the bits over that made sense to keep, and deleted the rest since it wasn't used.

Reply:
@andrey-p I think the max-width and overflow logic needs a div to be effective:
<img width="2058" alt="screen shot 2018-01-19 at 2 27 59 pm" src="https://user-images.githubusercontent.com/3652195/35167875-fd3b1e98-fd24-11e7-9379-da6521efafdb.png">

vs

<img width="2023" alt="screen shot 2018-01-19 at 2 27 40 pm" src="https://user-images.githubusercontent.com/3652195/35167879-0363d56c-fd25-11e7-84cb-68ec82e5b155.png">

Reply:
.url is currently a span, second screenshot is the current one

Reply:
Gah, sorry, I think I pushed without doublechecking. Lemme fix.